instagram/src/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
+import CommentSection from './components/CommentSection/CommentSection';
+
+import './App.css';
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state={
+      dummyData: dummyData,
```

Comment:
With ES6 Syntax you can just write `this.state = { dummyData }` any time the key is equal to the value